lib/elixir_google_scraper/account/user_tokens.ex

```diff
@@ -12,50 +12,41 @@ defmodule ElixirGoogleScraper.Accounts.UserToken do
   @change_email_validity_in_days 7
   @session_validity_in_days 60
 
-  schema "users_tokens" do
-    field :token, :binary
-    field :context, :string
-    field :sent_to, :string
-    belongs_to :user, ElixirGoogleScraper.Accounts.User
-
-    timestamps(updated_at: false)
-  end
-
-  @doc """
-  Generates a token that will be stored in a signed place,
-  such as session or cookie. As they are signed, those
-  tokens do not need to be hashed.
-  """
   def build_session_token(user) do
     token = :crypto.strong_rand_bytes(@rand_size)
 
     {token,
-     %ElixirGoogleScraper.Accounts.UserToken{token: token, context: "session", user_id: user.id}}
+     %UserToken{
+       token: token,
+       context: "session",
+       user_id: user.id
+     }}
   end
 
-  @doc """
-  Checks if the token is valid and returns its underlying lookup query.
-
-  The query returns the user found by the token.
-  """
   def verify_session_token_query(token) do
     query =
-      from token in token_and_context_query(token, "session"),
+      from(token in token_and_context_query(token, "session"),
         join: user in assoc(token, :user),
         where: token.inserted_at > ago(@session_validity_in_days, "day"),
         select: user
+      )
 
     {:ok, query}
   end
 
-  @doc """
-  Builds a token with a hashed counter part.
+  @spec build_email_token(atom | %{:email => any, :id => any, optional(any) => any}, any) ::
```

Comment:
Is this intended? 🤔

Reply:
My VS code plugin autogenerated it 🙈 , removed!